docs/blog/2021-community-kickoff.rst

```diff
@@ -0,0 +1,186 @@
+
+We hope you're okay,
+and we hope that the storm we have been engulfed in will soon pass.
+
+A heartfelt thanks
+------------------
+
+We had a lot of big plans for last year,
+but most of those had to be put on hold in March as we scrambled to move our community fully online.
+Getting to see all the happy faces at our in-person events is one of the largest motivators for people who organize events.
+Sadly this year, we weren't able to make that happen.
+
+2020 has shown that the love and care we put into the community will be returned back to us when needed.
+When we moved the conferences online,
+we were genuinely worried about the health and longevity of Write the Docs.
+As we noted in our `ticket choices`_ page for Portland,
+we had a lot of outstanding debts and we didn't know how much support we would get with our online conferences.
+It was easy to see a worst case scenario where our venues didn't refund us and people didn't support the online events.
+
+We are happy to report that we received overwhelming support from our vendors, sponsors, speakers, team, and attendees:
+
+* Most of our vendors refunded our down payments in full. Our A/V and captioning folks adapted perfectly to the new format.
+* Only 1 of our sponsors canceled because of the transition to a virtual event.
+* Our speakers pulled together to record videos and handle live online Q&A.
+* Our team pulled together to run virtual events that exceeded our expectations for what those events could be.
+* Our attendees, most importantly, made our virtual events *feel* like our conferences, with a supportive and generous spirit.
+
+2020 has been a hard year,
+but we wanted to take this chance to give thanks to those who made 2020 a special year.
+We feel incredibly lucky to be able to support this wonderful community,
+and we look forward to making 2021 the best year yet,
+with your help.
+
+.. _ticket choices: https://www.writethedocs.org/conf/portland/2020/ticket-choices/
+
+Looking forward
+---------------
+
+One of the largest things we noticed in 2020 was how much more support our community needed.
+As we scrambled to move our events online,
+people were also adjusting to working remotely for the first time.
+We saw lots of activity in our Slack network,
+as people were trying to connect online.
+Meetups slowly adjusted,
+trying to move online and keep some momentum going.
+
+Write the Docs itself was able to provide similar value as in the past,
+but we didn't have the resources to adjust to the changing world.
+The profits from our conferences pay for everything we do,
+which constrains our ability to adjust to events as they happen.
+
+We are planning a way for our community to support the organization directly.
+We are proud that all the content that Write the Docs produces is free to everyone,
+and we don't plan to change that any time soon.
+The goal for 2021 is to be able to provide additional support to our community,
+paid for by members in that community.
```

Comment:
```suggestion
funded by members in that community, who are able to contribute.
```
The second addition feels a bit wonky wording, but I feel we should help people who can't contribute feel just as welcome in the future. Maybe the previous sentence covers that enough though.